hazelcast/src/main/java/com/hazelcast/spi/impl/operationexecutor/classic/ClassicOperationExecutor.java

```diff
@@ -108,6 +112,10 @@ public ClassicOperationExecutor(GroupProperties properties,
                 + partitionOperationThreads.length + " partition operation threads.");
     }
 
+    private void registerThreadSensors(OperationThread operationThread) {
```

Comment:
Good spotting. That is old code before the probe annotation scanning was added I think.